### ISSUE TITLE
trigger with "click" string in type of the event

### ISSUE THREAD

**Issue by vincentbeutin** [webpro/DOMtastic#50]
hi,

if i trigger an event `test_click`, a `Mouseevent` is fired and not a `CustomEvent` and params are empty in the listener.

```
// Not working
sp$('body').trigger('test_click', {'param1':'value1'});
window.addEventListener("test_click",function(event, toto, tata){
        console.log('param1 = '+event.detail.param1);
});

// Working
 sp$('body').trigger('test_clck', {'param1':'value1'});
window.addEventListener("test_clck",function(event, toto, tata){
       console.log('param1 = '+event.detail.param1);
});
```

version = '0.10.3'

**Comment by webpro** [comment]
Hi, @vincentbeutin. Yes, DOMtastic tries to detect the appropriate event type to be fired, and dispatches a `MouseEvent` if the event `type` matches this regular expression: `/^(?:mouse|pointer|contextmenu)|click/`. The `detail` property is set by DOMtastic, but to no effect, since it's read-only for a `MouseEvent`.

An actual mouse event (by a real user) also has `details` set by the browser.

I agree this should at least be documented better.

**Comment by webpro** [comment]
Fixed in a70b2f6b631a4d9ad5b6f3ad007491941967097b
